src/gui/properties/downloadedpiecesbar.cpp

```diff
@@ -172,7 +172,11 @@ void DownloadedPiecesBar::clear()
 
 QString DownloadedPiecesBar::simpleToolTipText() const
 {
-    return tr("White: Missing pieces") + '\n'
-           + tr("Green: Partial pieces") + '\n'
-           + tr("Blue: Completed pieces") + '\n';
+    const QString colorBoxBorderColor = palette().color(QPalette::ToolTipText).name();
+    const QString rowHTML = QString::fromLatin1("<tr><td width=20 bgcolor='%1' style='border: 1px solid \"%2\";'><td> <td>%3</td></tr>");
```

Comment:
Shouldn't it be?
```html
<tr><td width=20 bgcolor='%1' style='border: 1px solid \"%2\";'>%3</td></tr>
```
or
```html
<tr><td width=20 bgcolor='%1' style='border: 1px solid \"%2\";'></td><td>%3</td></tr>
```

Reply:
current 
![image](https://user-images.githubusercontent.com/34717789/90330093-de57bf80-dfc7-11ea-9558-4fe17d5869c8.png)

>` <tr><td width=20 bgcolor='%1' style='border: 1px solid \"%2\";'>%3</td></tr>`

![image](https://user-images.githubusercontent.com/34717789/90330106-f596ad00-dfc7-11ea-8370-3ee7b158c4fa.png)


> `<tr><td width=20 bgcolor='%1' style='border: 1px solid \"%2\";'></td><td>%3</td></tr>`

![image](https://user-images.githubusercontent.com/34717789/90330131-0c3d0400-dfc8-11ea-8956-eb930e41c4e5.png)

Reply:
OK.

Reply:
oops, another thing I missed.
This line: `<td width=20 bgcolor='%1' style='border: 1px solid \"%2\";'><td>`, the second `<td>` should be closed properly: `</td>`, right?